com.unity.netcode.gameobjects/Runtime/Transports/UTP/UnityTransport.cs

```diff
@@ -256,10 +259,14 @@ private static NetworkEndpoint ParseNetworkEndpoint(string ip, ushort port, bool
                 if (!NetworkEndpoint.TryParse(ip, port, out endpoint, NetworkFamily.Ipv4) &&
                     !NetworkEndpoint.TryParse(ip, port, out endpoint, NetworkFamily.Ipv6))
                 {
+#if HOSTNAME_RESOLUTION_AVAILABLE && UTP_TRANSPORT_2_4_ABOVE
+                    return default;
```

Comment:
Do we want to respect the silent parameter?

Reply:
Actually you make a good point. _(good catch)_
The client uses this method once, but the server uses it in a few places. 
It needs a bit of a refactor... perhaps a bool signifying whether it is parsing it for a server address, which in that case we should honor the silent flag, or if it is a client... which in that case we might do a Fqdn check if it reaches that point.